mne/digitization/base.py

```diff
@@ -103,3 +103,18 @@ def __eq__(self, other):  # noqa: D105
             return False
         else:
             return all([ss == oo for ss, oo in zip(self, other)])
+
+    def plot(self, info=None, trans=None, subject=None, subjects_dir=None,
+             surfaces='head', coord_frame='head',
+             meg=None, eeg='original',
+             dig=False, ecog=True, src=None, mri_fiducials=False,
+             bem=None, seeg=True, show_axes=False, fig=None,
+             interaction='trackball', verbose=None):
```

Comment:
@GuillaumeFavelier if you require all the same things as plot_alignment then I don't see the point.

dig.plot() should just work.
or if you wnat to couple with a subject:
dig.plot(trans, subject, subjects_dir)